tests/equation_bcs/test_bcs.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from firedrake import *
+
+
+@pytest.mark.parallel
+def test_bc_on_sub_sub_domain():
+
+    # Solve a vector poisson problem
+
+    mesh = UnitSquareMesh(32, 32)
+
+    V = VectorFunctionSpace(mesh, "CG", 4)
```

Comment:
Is degree 4 necessary to provoke the error, or will 1 do?

Reply:
Fixed! Let's use 1.